pl_examples/basic_examples/lightning_module_template.py

```diff
@@ -214,17 +214,17 @@ def __dataloader(self, train):
 
     @pl.data_loader
     def train_dataloader(self):
-        print('training data loader called')
+        self.logger.info('training data loader called')
```

Comment:
this is not very saved with `self.logger = None` in `LightningModule`

Reply:
ah indeed. in that case exposing the logger is becoming much harder 🌚

Reply:
You may try change `self.logger = logging.getLogger()`